analysis_plan.md

```diff
@@ -0,0 +1,25 @@
+Comparison of SARS-CoV & SARS-CoV-2 Viruses from China
+------------------------------------------------------
+------------------------------------------------------
+
+1. Search literature for nucleotide conbinations that differ in SARS-CoV & SARS-CoV-2 viruses
+
+2. Decide which DNA nucleotide combinations add to the Dictionary
+
+3. Use ```function kmercount(sequence, k)``` to calculate the numerical difference of the nucleotide 
+
+conbinations of interest in the DNA of SARS-CoV & SARS-CoV-2 viruses
+
+4. Present the difference with the bar graph
+
+Comparison of SARS-CoV-2 viruses from China & from Brazil
+---------------------------------------------------------
+---------------------------------------------------------
+
+1. Use ```function basecomposition(sequence)``` to count number of A,T,G,C nucleotides in the DNA of 
```

Comment:
Base composition doesn't really tell us all that much, it's a very crude metric. Consider doing some alignments, or compare the effects of different kmer lengths